src/components/ui/particles.tsx

```
@@ -0,0 +1,394 @@
+    particles: null,
+  });
+
+  // Create geometry data
+  const createGeometryData = useCallback((count: number, colors: string[]) => {
+    const positions = new Float32Array(count * 3);
+    const randoms = new Float32Array(count * 4);
+    const colorData = new Float32Array(count * 3);
+    const palette = colors && colors.length > 0 ? colors : defaultColors;
+
+    for (let i = 0; i < count; i++) {
+      let x: number, y: number, z: number, len: number;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)